image.c

```diff
@@ -403,7 +403,7 @@ bool img_load_webp(const fileinfo_t *file, Imlib_Image *fframe, img_t *img)
 			imlib_image_set_format("webp");
 			/* Get an iterator of this frame - used for frame info (duration, etc.) */
 			WebPDemuxGetFrame(demux, img->multi.cnt+1, &iter);
-			imlib_image_set_has_alpha((flags & ALPHA_FLAG) > 0);
+			imlib_image_set_has_alpha(flags & ALPHA_FLAG);
```

Comment:
This is of course making assumptions that `imlib_image_set_has_alpha` supports values other than 0 and 1.

Reply:
The document only mentions `0` and `1`, however the function simply does an if check on the param. So it should work.
But I guess it's best to be safe, I'll update this.